doc/nrf/external_comp/nrf_cloud.rst

```diff
@@ -27,13 +27,70 @@ To read more about nRF Cloud, see the `nRF Cloud`_ website and the `nRF Cloud do
 You can use the services offered by nRF Cloud in the following scenarios:
 
 * Device connected to nRF Cloud over CoAP.
+
   The services can be used from nRF Cloud.
+
 * Device connected to nRF Cloud over MQTT.
+
   The services can be used from nRF Cloud.
+
 * Device connected to nRF Cloud over MQTT, with a customer-developed website or application that interacts with the `nRF Cloud REST API`_ to display device data and manage it in a customized way.
+
 * Device connected to nRF Cloud over REST, interacting using the `nRF Cloud REST API`_.
+
 * Device connected to a customer cloud service in a suitable manner. The services can be used from the customer cloud service that communicates over REST to the nRF Cloud REST API in a proxy configuration.
 
+Choosing a device ID format: prefix-IMEI, device UUID, or other
+***************************************************************
+
+nRF Cloud supports a variety of device ID formats:
+
+* A prefix plus IMEI
+
+  This is reserved for Nordic development kits or evaluation platforms when the prefix is ``nrf-``.
+
+* The device UUID
+
+  This is a guaranteed unique ID stored in the modem.
+
+* Compile-time
+
+  A fixed device ID compiled into the firmware.
+
+* Run-time
+
+  A device ID generated by the application in an application-specific way.
+
+* HW-ID
+
+  A device ID generated from one of a selection of hardware IDs.
+
+Offical Nordic development kits such as the nRF9160DK have a sticker on the back indicating the device's IMEI and PIN.
+They are preprovisioned with JITP certificates so they can be added easily to nRF Cloud using the device ID ``nrf-IMEI`` and PIN.
+Customers using custom hardware cannot use ``nrf-<IMEI>`` style device ids.
+Names that start with ``nrf-`` and their PIN or HWID are checked against a database of genuine Nordic development kits and Thingy:91s.
+If not found in the database, the device cannot connect.
+Choose your own PIN when creating creating JITP credentials for a custom device.
+
+The `device_credentials_installer.py`_ script uses the device UUID format by default.
+See these command line options to use other device ID formats:
+
+* --id_str ID_STR
+
+  Device ID to use instead of UUID. Will be a prefix if used with --id_imei (default: )
+
+* --id_imei
+
+  Use IMEI for device ID instead of UUID. Add a prefix with --id_str (default: False)
+
+Your firmware's device ID configuration must align with the options used when executing device_credentials_installer.py.
+See: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/nrf_cloud.html#configuration-options-for-device-id
+
+The ``device_device_credentials_installer.py`` script automatically connects over ``USB`` or ``RTT`` to a device and installs the certificates in the modem -- no need to use nRF Connect for Desktop’s Cellular Monitor.
```

Comment:
```suggestion
The :file:`device_credentials_installer.py` script automatically connects over USB or RTT to your device and installs the certificates in the modem.
```
No need to dunk on our own offerings.

Reply:
I didn't think I was dunking, but it's OK to change it. We have found some customers try to mix different methods like this, so I was really just trying to prevent this in the future.